### ISSUE TITLE
On a Form that is enhanced with Aria invalid, NVDA does not read the next Formfield after announcing the error alert.

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#3807]
**Reported by daniele.corciulo on 2014-01-23 08:53**
on this example http://www.marco-zehe.de/examples/Tutorial_aria-invalid_and_role_alert.html there is a Problem. If you do not fill in the required information and press tab to go to the next field, NVDA announces correctly the alert warning. The Problem is, that it doesn't announc, that you are also in the next field. You hear only the errormessage without feedback of the current Cursorposition.
I expect to hear after the errormessage my new current cursorposition.

**Comment by leonardder** [comment]
Interesting problem, which I can reproduce. The point is, when you move from name to e-mail address, there is an alert triggered, but the alert is shown after the e-mail address edit box gains focus. Thus, the edit box is read, but the alert interrupts reading of the newly focused control.

cc @marcozehe

**Comment by jcsteh** [comment]
Probably requires the queuing part of speech refactor (#4877).

**Comment by jcsteh** [comment]
P3 because it's annoying, but there's a workaround: just press NVDA+tab to report the focus.

**Comment by Adriani90** [comment]
This is not reproducible in Firefox 64.0. But I can reproduce it in Google Crome 71.0. In Internet Explorer and Edge, notifications are not reported at all when tabbing around.

**Comment by Adriani90** [comment]
@daniele.corciulo the link in the description is not working anymore. Could you please post a new one? Thanks.

**Comment by Adriani90** [comment]
No updates by the initial author since years and the link does not work anymore. Closinga s abandoned.
